lib/src/main/java/co/nimblehq/alpine/lib/model/nfc/AdditionalDocumentDetails.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.alpine.lib.model.nfc
```

Comment:
Can it be just `model` without sub-directory `nfc`?

Reply:
@Wadeewee or how about moving `MrzInfo` to the `model.mrz` subdirectory? 😆